templates/default-no-html.md

```diff
@@ -1,6 +1,9 @@
 # Welcome to <%= projectName %> 👋
-<% if (projectVersion) { -%>
-![Version](https://img.shields.io/badge/version-<%= projectVersion %>-blue.svg?cacheSeconds=2592000)
+<% if (projectVersion.type === "static" && projectVersion.data) { -%>
```

Comment:
As mentioned before, the rendering should depends on a boolean property instead of a type